src/js/providers/flash.js

```diff
@@ -103,7 +103,25 @@ define([
         }
 
         _.extend(this, Events, {
+                init: function(item) {
+                    // if preload is none or autostart is true, do nothing
+                    if (item.preload === 'none' || _playerConfig.autostart) {
+                        return;
+                    } else {
+                        _item = item;
+
+                        // if we just call flashCommand with _swf undefined, init will go into commandQueue
+                        if (_swf) {
+                            _flashCommand('init', item);
+                        } else {
+                            this.once('swfDefined', function() {
```

Comment:
This actually happens, and init goes into "queueCommand", which is ran when we are trying to load.